src/Arcomage/Arcomage.cpp

```diff
@@ -188,8 +188,8 @@ bool arcomage_mouse::Update() {
     return true;
 }
 
-bool arcomage_mouse::Inside(Rect *pXYZW) {
-    return (x >= pXYZW->x) && (x <= pXYZW->z) && (y >= pXYZW->y) && (y <= pXYZW->w);
+bool arcomage_mouse::Inside(Recti *pXYZW) {
+    return (x >= pXYZW->x) && (x <= pXYZW->x + pXYZW->h) && (y >= pXYZW->y) && (y <= pXYZW->y + pXYZW->w);
```

Comment:
*pXYZW -> *pXYWH on variables names in a few places

Reply:
Renamed to pRect